tests/storage/disk_preallocation/utils.py

```
@@ -0,0 +1,48 @@
+        f"'{preallocation_requested_annotation}' should be '{status}'"
+    )
+
+
+def assert_preallocation_annotation(pvc: PersistentVolumeClaim, res: str) -> None:
+    preallocation_annotation = f"{NamespacedResource.ApiGroup.CDI_KUBEVIRT_IO}/storage.preallocation"
+    assert pvc.instance.metadata.annotations.get(preallocation_annotation) == res, (
+        f"'{preallocation_annotation}' should be '{res}'"
+    )
+
+
+def wait_for_cdi_preallocation_enabled(cdi_config: CDIConfig, expected_value: bool) -> None:
```

Ruff (0.15.17)

[warning] Boolean-typed positional argument in function definition

(FBT001)